crates/sui-core/src/authority_aggregator.rs

```diff
@@ -159,6 +161,19 @@ impl AuthAggMetrics {
                 registry,
             )
             .unwrap(),
+            total_equivocation_detected: register_int_counter_with_registry!(
+                "total_equivocation_detected",
+                "Total number of equivocations that are detected",
```

Comment:
Maybe call it client-equivocation/client-double-spend to differentiate from the much more dangerous validator equivocation? In general equivocation rings all kind of danger alarms in the head of people building consensus engines and this is really not that dangerous.